app/models/user.rb

```diff
@@ -0,0 +1,22 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Please remove this configuration as you have it duplicated below, the one below is updated configuration  👍🏿 
`  devise :database_authenticatable, :registerable,
         :recoverable, :rememberable, :validatable, :confirmable`